### ISSUE TITLE
Rename files displays a $Type in filenames instead of the edition

### ISSUE THREAD

**Issue by Frankthetankk** [mylar3/mylar3#421]
**Describe the bug**
In the 'File Format' field under mylar's Settings > Advanced Settings. I am using `$Type` as a setting. When I go to any series and when I use the 'Rename Files' button in mylar, the files in the series folder will have a `$Type` in the file name instead of the actual edition name. For example `Batman Rebirth 1 (August 2016) $Type.cbz` instead of `Batman Rebirth 1 (August 2016) One-Shot.cbz`.  

**Environment:**
- OS: Windows10 VM
- Branch master
- git (build 3af6bb19fec0d2426d076ae574a3523a7a12ad6e)
- Python Version : 3.7.6
- Language : en_US.cp1252
- installed via [Install Mylar Windows for Usenet Comics](https://www.htpcguides.com/install-mylar-windows-usenet-comics/) but changed info to mylar3.

**Additional information**
Using `$Type` in the 'Folder Format' field works ok. For example, the folder is formated to `DC Comics - Batman Rebirth (2016) One-Shot`

**Comment by evilhero** [comment]
I don't believe ```$Type``` was ever setup to work with File Formatting, only Folder Formatting. This would be due (most likely) to the filename parser not being able to properly parse the given ```$Type``` from the filename cleanly (as everyone will have it different).

I know it says it exists in the Info hover-over for File Formatting, but I'll have to double-check that it actually is present as it might have been accidentally placed there at one point and never corrected.

**Comment by Frankthetankk** [comment]
> I know it says it exists in the Info hover-over for File Formatting

This is exactly why I reported this issue. Thanks
